Makefile

```diff
@@ -67,7 +67,7 @@ CCTZ_OBJS =			\
 TOOLS = time_tool
 EXAMPLES = classic epoch_shift hello example1 example2 example3 example4
 
-all: $(TESTS) $(TOOLS) $(EXAMPLES)
+all: $(TESTS) $(TOOLS) $(EXAMPLES) $(CCTZ_SHARED_LINK)
```

Comment:
It's not. Was just so that the shared library ended up in the build folder when doing `make all`. Will remove it.